### DISCUSSION TITLE
become a github sponsor

### DISCUSSION THREAD

**Discussion by one7two99** [KRTirtho/spotube#475] [Q&A]
hello and thanks for your work.
would it be possible to become a github sponsor, which  allow other people to see who is "sponsoring" your project. May be better and more visible compared to the buy me a coffee page from you:
https://www.buymeacoffee.com/krtirtho
it's a shame that you have much more stars than coffee. maybe enabling the github sponsor feature will help?

**Comment by KRTirtho** [comment]
Unfortunately, the GitHub sponsor isn't supported in my region so I'm bound to use 3rd party services 😞
